src/Take.Elephant.Sql/ExpirableKeySqlMap.cs

```diff
@@ -157,4 +163,69 @@ private static string InjectSqlFilter(string sql, string filter)
             }
         }
     }
+
+    /// <summary>
+    /// Implements a new behavior for DbConnection with a custom parameter.
+    /// </summary>
+    internal class ExpirationDbConnection : DbConnection
+    {
+        private readonly DbConnection _dbConnection;
+        public ExpirationDbConnection(DbConnection dbConnection)
+        {
+            _dbConnection = dbConnection;
+        }
+
+        public override string ConnectionString { get => _dbConnection.ConnectionString; set => _dbConnection.ConnectionString = value; }
+
+        public override string Database => _dbConnection.Database;
+
+        public override string DataSource => _dbConnection.DataSource;
+
+        public override string ServerVersion => _dbConnection.ServerVersion;
+
+        public override ConnectionState State => _dbConnection.State;
+
+        public override void ChangeDatabase(string databaseName)
+        {
+            _dbConnection.ChangeDatabase(databaseName);
+        }
+
+        public override void Close()
+        {
+            _dbConnection.Close();
+        }
+
+        public override void Open()
+        {
+            _dbConnection.Open();
+        }
+
+        protected override DbCommand CreateDbCommand()
+        {
+            var command = _dbConnection.CreateCommand();
+            var parameter = command.CreateParameter();
+            parameter.ParameterName = "@ExpirableKeySqlMap_ExpirationDate";
```

Comment:
Extract this name to a constant, that ExpirationDatabaseDriver can use when injecting its filter.

Reply:
Done